### DISCUSSION TITLE
AIT National

### DISCUSSION THREAD

**Discussion by booziebatsymbol** [iptv-org/community#814] [Show and tell]
### Channel ID

AITNational.ng

### Stream URL

https://www.youtube.com/watch?v=g-SjiuYZYUU&ab_channel=AITLiveStreaming

### Notes

[Not 24/7]

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/AITLiveStreaming1/live

They split programs on stream, to probably put them on their account as a replay after.

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/AITLiveStreaming1/live (offline)
<img width="693" alt="image" src="https://user-images.githubusercontent.com/7253922/232234945-d1a66e79-36ff-4313-8897-28ffd3967fde.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ng.m3u#L10-L11
